Xamarin.Forms.Platform.Tizen/Renderers/DefaultRenderer.cs

```diff
@@ -15,4 +16,92 @@ protected override void OnElementChanged(ElementChangedEventArgs<VisualElement>
 			base.OnElementChanged(e);
 		}
 	}
-}
\ No newline at end of file
+
+	// Following Shapes dummy renderers can be replaced to real renderers,
+	// when `InitializationOptions.UseSkiaSharp` value is set to true in an application.
+	// ```
+	// var option = new InitializationOptions(app)
+	// {
+	//         UseSkiaSharp = true
+	// };
+	// Forms.Init(option);
+	// ```
+
+	public class EllipseRenderer : VisualElementRenderer<Ellipse>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Ellipse> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+
+	public class LineRenderer : VisualElementRenderer<Line>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Line> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+
+	public class PathRenderer : VisualElementRenderer<Path>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Path> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+
+	public class PolygonRenderer : VisualElementRenderer<Shapes.Polygon>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Shapes.Polygon> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+
+	public class PolylineRenderer : VisualElementRenderer<Polyline>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Polyline> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+
+	public class RectangleRenderer : VisualElementRenderer<Shapes.Rectangle>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<Shapes.Rectangle> e)
+		{
+			if (NativeView == null)
+			{
+				var control = new ELayout(Forms.NativeParent);
+				SetNativeView(control);
+			}
+			base.OnElementChanged(e);
+		}
+	}
+}
```

Comment:
Not a big deal but, it would be better to simplify by removing redundant code.

Reply:
True. I just removed the redundant code and tested it again